sql/src/main/java/io/crate/execution/jobs/JobSetup.java

```diff
@@ -129,7 +129,7 @@ public JobSetup(Settings settings,
                     MapSideDataCollectOperation collectOperation,
                     ClusterService clusterService,
                     NodeJobsCounter nodeJobsCounter,
-                    CrateCircuitBreakerService breakerService,
+                    CrateCircuitBreakerService circuitBreakerService,
```

Comment:
Using a `Function<String, CircuitBreaker>` here would also perfectly fit imho (decoupling, eliminates test mocking).

Reply:
Here we cannot use the Function as this kind of injection is not supported by Guice.

Reply:
ah sry sure, didn't see that this is injected